pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md

```diff
@@ -50,16 +50,17 @@ The air-gapped bundle includes the following packages and tooling
 - Docker registry image
 
 1.  Download the air-gapped bundle.
+<p class="message--important"><strong>NOTE: </strong>Enter in the appropriate version in the $VERSION field.</p>
```

Comment:
Sorry, my previous suggestion for the VERSION wasn't very clear...  My idea was to add another step to this procedure something like we do here in the docs:   https://docs.d2iq.com/dkp/kommander/2.0/install/networked/#prerequisites, where we tell the user to define a variable we use to eliminate some of repetitive typing later on, and also to make for less things to update when we spin a new version.

So my suggestion would be to do something like this:  (but put the export in the ```sh code block, I couldn't get the suggestion thing to accept nested backtick sections)

```suggestion
Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing - you can find the correct Kubernetes version by checking the release notes for the release you are installing.

export VERSION=1.21.6

```

Reply:
If we are doing Version for K8s - should we change the env var name to: 

export KUBERNETES_VERSION=1.21.6

And then change that var down below, too?